src/com/hms_networks/americas/sc/datapoint/package.html

```diff
@@ -3,7 +3,7 @@
 Data point classes for storing information related to
 and the contents of data points from the Ewon historical log.
 
-@version 1.0
+@version 1.1
```

Comment:
Do we need to rev the version considering nothing has really used these yet and you haven't Tagged the repo?

Reply:
reverted